tests/unit/windows/test_winapilayer.cpp

```diff
@@ -160,10 +160,21 @@ TEST(WinApiLayer, GetFriendlyName) {
   }
 }
 
+TEST(WinApiLayer, GetFriendlyNameFromInvalidPath) {
```

Comment:
The first part of `TEST(WinApiLayer, GetFriendlyNameFromInvalidPath)` and `TEST(WinApiLayer, GetDisplayNameInvalidPath)` are identical. Perhaps a fixture should be made instead?

Reply:
Added some fixtures where it makes sense, some stuff was put into global consts instead. Finally, seasoned everything with some preprocessor macro magic to make test cases more readable.